ludwig/features/image_feature.py

```diff
@@ -81,7 +81,7 @@ def add_feature_data(
                 )
             feature['should_resize'] = True
             set_default_value(
-                feature['preprocessing'], 
+                preprocessing_parameters, 
```

Comment:
This should be added in preprocessing defaults https://github.com/uber/ludwig/blob/6f60f10cf3c30f4f5c013bb3f6e8961b074dcdcf/ludwig/features/image_feature.py#L41 because that dictionary is merged with what the user provides, by doing this i believe we are overriding the user provided info.

Reply:
Thx for catching this, updated.